_data/tools.yml

```diff
@@ -54,6 +301,22 @@
       url: https://trino.io/docs/current/client/cli.html
     - urltext: User guide
       url: https://docs.starburst.io/data-consumer/clients/cli.html
+- name: Clickhouse
+  anchor: clickhouse
+  category: data-source
+  logo: /assets/images/logos/clickhouse.png
+  logosmall: /assets/images/logos/clickhouse-small.png
+  description: |
+    ClickHouse is the fastest and most resource efficient open-source database
+    for real-time apps and analytics.
+
+    Use a Clickhouse database as a data source in Trino by configuring a catalog
```

Comment:
```suggestion
    Use a Clickhouse real-time database as a data source in Trino by configuring a catalog
```

Reply:
I worked this better into the above paragraph .. this connection sentence should be similar across all of them